src/collection/views/CollectionDetail.tsx

```diff
@@ -101,6 +103,88 @@ const CollectionDetail: FunctionComponent<CollectionDetailProps> = ({
 	// Mutations
 	const [triggerCollectionDelete] = useMutation(DELETE_COLLECTION);
 
+	const checkPermissions = useCallback(async () => {
+		try {
+			const rawPermissions = await Promise.all([
```

Comment:
Deze logica ook naar een service eventueel?

Reply:
het hangt ervan af hoe je de code indeling wil doen. Ik doe vooral in een service als ik het nog ergens anders ook nodig heb.

@andrych @benjaminnaesen wat denken jullie

Reply:
Ik doe meestal alles van crud stuff in services, vooral om lange files voor de views en dergelijke te vermijden. Maar zolang dat meevalt en enkel op 1 plek gebruikt wordt mag dat best in 1 file blijven.